webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -353,16 +353,17 @@ object DisplayNode extends Loggable {
    * Should be used with jsInit(dn:String, softIds:Seq[SoftwareUuid], salt:String="")
    */
   def showPannedContent(
-      nodeAndGlobalMode: Option[(NodeInfo, GlobalPolicyMode)],
-      sm:                FullInventory,
-      inventoryStatus:   InventoryStatus,
-      salt:              String = ""
-  ): NodeSeq = {
-    val jsId      = JsNodeId(sm.node.main.id, salt)
+      nodeFact:   NodeFact,
+      globalMode: GlobalPolicyMode,
+      salt:       String = ""
+  )(implicit qc:  QueryContext): NodeSeq = {
+    val jsId      = JsNodeId(nodeFact.id, salt)
     val detailsId = htmlId(jsId, "details_")
+    val sm        = nodeFact.toFullInventory
```

Comment:
We keep the mapping with `FullNodeInventory` because that part is low risk, but big => it can be done in latter iteration.